WeakAurasOptions/BuffTrigger2.lua

```diff
@@ -160,6 +173,23 @@ local function CreateNameOptions(aura_options, data, trigger, size, isExactSpell
         end
         return icon and tostring(icon) or "", 18, 18
       end
+
+      aura_options[iconOption].func = function()
```

Comment:
i don't love that this is such a hidden feature, though i can't think of a way to improve that without tearing holes in how we layout these options.